### DISCUSSION TITLE
Why Data is not showing in before while creating an action

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#13014] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

@leandrocfe do you know?
```
 Undefined array key "status_to_id"
 CreateAction::make()
                    ->before(function (CreateAction $action, RelationManager $livewire, array $data) {
                     $toID = $data['status_to_id'];

}
```

**Comment by zeeshantariq08** [comment]
it was workling before but today it is not working

**Comment by leandrocfe** [comment]
Let me check

**Reply by zeeshantariq08** [reply]
thank you

**Reply by zeeshantariq08** [reply]
@leandrocfe have you checked?

**Comment by bzy107** [comment]
Isn't this fixed in this pull request?

https://github.com/filamentphp/filament/pull/12987

**Reply by bzy107** [reply]
It has been released in v3.2.84.

**Reply by leandrocfe** [reply]
@zeeshantariq08 , check this 👆

**Reply by AFSDARIF** [reply]
thanks, solved. just run composer update
